custom_components/tahoma/climate_devices/stateless_exterior_heating.py

```diff
@@ -0,0 +1,97 @@
+"""Support for Atlantic Electrical Heater IO controller."""
+import logging
+from pprint import pformat
+from typing import List, Optional
+
+from custom_components.tahoma.climate import SUPPORT_MY
```

Comment:
This will not work in all situations I believe. Could you use the notation without `custom-components`.

Reply:
you're write! I let this slip through my checks...